DSCResources/MSFT_xSQLServerSetup/MSFT_xSQLServerSetup.psm1

```diff
@@ -176,6 +180,18 @@ function Get-TargetResource
         $SQLSvcAccountUsername = (Get-WmiObject -Class Win32_Service | Where-Object {$_.Name -eq $DBServiceName}).StartName
         $AgtSvcAccountUsername = (Get-WmiObject -Class Win32_Service | Where-Object {$_.Name -eq $AgtServiceName}).StartName
         $FullInstanceID = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Microsoft SQL Server\Instance Names\SQL' -Name $InstanceName).$InstanceName
+
+        #test if Replication sub component is configured for this instance
+        Write-Verbose "Detecting replication feature"
+        Write-Verbose "Reading registry key HKLM:\SOFTWARE\Microsoft\Microsoft SQL Server\$FullInstanceID\ConfigurationState"
+        $IsReplicationInstalled = (Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Microsoft SQL Server\$FullInstanceID\ConfigurationState").SQL_Replication_Core_Inst
```

Comment:
Please update local variable names to start with a lowercase. See the guidelines: https://github.com/PowerShell/DscResources/blob/TravisEz13-patch-1/StyleGuidelines.md#variable-names-should-use-camelcase